src/main_ghostty.zig

```diff
@@ -96,11 +96,13 @@ pub fn main() !MainReturn {
     }
 
     // Create our app state
-    var app = try App.create(alloc);
-    defer app.destroy();
+    var app: App = undefined;
+    defer app.deinit();
+    try app.init(alloc);
```

Comment:
If `init` fails, `app.deinit()` will be called on an undefined object. The `defer` should come after the `init`.

```suggestion
    try app.init(alloc);
    defer app.deinit();
```

Reply:
Fixed.